dpnp/tests/test_nanfunctions.py

```diff
@@ -505,86 +501,78 @@ class TestNanProd:
     @pytest.mark.parametrize("axis", [None, 0, 1, -1, 2, -2, (1, 2), (0, -2)])
```

Comment:
Should we combine `TestNanProd` and `TestNanSum` to one `TestNanProdSum` class with parametrization by a function name?